### PR TITLE
VACMS-19013: Adding Vet Center services

### PR DESCRIPTION
**Pull request by omahane**
## Description

Relates to #19013

## Testing done
Manually

## Screenshots
Screenshot 2024-08-21 at 11 35 15 AM

## QA steps
As a content admin
1. Go to the [Facility services content audit](https://pr19014-ea1hvehzgnmspuepp6d2g37qkxuzjahq.ci.cms.va.gov/admin/content/audit/facility-services)
   - [x] Confirm that you can filter by "Vet Center - Facility Service" in the **Content type** exposed filter
   - [x] Choose "Vet Center - Facility Service" in the **Content type** exposed filter
   - [x] Click **Filter**
   - [x] Confirm that Vet Center - Facility Services are returned in the view
2. Then, check that the form hasn't regressed
   - [x] Choose "VAMC Facility Health Service" in the **Content type** exposed filter
   - [x] Click **Filter**
   - [x] Confirm that VAMC Facility Health Services are returned in the view

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

**Comment by laflannery** [comment]
@omahane When I filter, I'm seeing that every Vet Center service is showing at 0 for the "Service location" column.  I feel like it is because it doesn't use the specific Service location paragraph right? Everything else looks good but I wanted to confirm my understanding here

**Comment by omahane** [comment]
> @omahane When I filter, I'm seeing that every Vet Center service is showing at 0 for the "Service location" column. I feel like it is because it doesn't use the specific Service location paragraph right? Everything else looks good but I wanted to confirm my understanding here

That's right. No service locations in Vet Centers. 

> Sorry I found one more thing - I can't seem to be able to use the Facility name filter to find Vet Center services: ![image]

This should now be fixed.

**Comment by laflannery** [comment]
QA Steps tested and working well and confirmed the issue from my previous comment is resolved!

**Comment by davidmpickett** [comment]
Followed QA steps and looks good

File: config/sync/views.view.facility_services.yml [hunk 1]
```diff
@@ -608,6 +608,7 @@ display:
         type: full
         options:
           offset: 0
+          pagination_heading_level: h4
```
Comment (jv-agile6) [inline config/sync/views.view.facility_services.yml:611]: Added h4 pagination heading

File: config/sync/views.view.facility_services.yml [hunk 2]
```diff
@@ -642,7 +643,7 @@ display:
             content_admin: content_admin
             administrator: administrator
       cache:
-        type: tag
+        type: none
```
Comment (jv-agile6) [inline config/sync/views.view.facility_services.yml:646]: Removed tag

File: config/sync/views.view.facility_services.yml [hunk 3]
```diff
@@ -9208,6 +9209,7 @@ display:
             health_care_local_health_service: health_care_local_health_service
             vha_facility_nonclinical_service: vha_facility_nonclinical_service
             vba_facility_service: vba_facility_service
+            vet_center_facility_health_servi: vet_center_facility_health_servi
```
Comment (jv-agile6) [inline config/sync/views.view.facility_services.yml:9212]: Added vet center facility health services

File: config/sync/views.view.facility_services.yml [hunk 4]
```diff
@@ -9241,7 +9243,7 @@ display:
           id: title_1
           table: node_field_data
           field: title
-          relationship: field_facility_location
+          relationship: none
```
Comment (jv-agile6) [inline config/sync/views.view.facility_services.yml:9246]: removed field_facility_location from relationship

File: config/sync/views.view.facility_services.yml [hunk 5]
```diff
@@ -9308,6 +9315,7 @@ display:
             health_care_local_health_service: health_care_local_health_service
             vha_facility_nonclinical_service: vha_facility_nonclinical_service
             vba_facility_service: vba_facility_service
+            vet_center_facility_health_servi: vet_center_facility_health_servi
```
Comment (jv-agile6) [inline config/sync/views.view.facility_services.yml:9318]: added vet center to facility health services

**Review by jv-agile6 — APPROVED** [review]
Code for adding vet center facility services to exposed filter and content type was reviewed for logic and form. No defects found. 🟢

**Pull request merged**